libexec/tgenv-list-remote

```diff
@@ -12,9 +22,15 @@ fi
 link_release="https://api.github.com/repos/gruntwork-io/terragrunt/tags?per_page=1000"
 print=$(curl --tlsv1.2 -sf $link_release)
 
+for (( page=2; page <= pages; page++))
+do
+    link_release="https://api.github.com/repos/gruntwork-io/terragrunt/tags?per_page=1000&page=${page}"
+    print+=$(curl --tlsv1.2 -sf $link_release)
+done
+
 return_code=$?
 if [ $return_code -eq 22 ];then
-  warn_and_continue "Failed to get list verion on $link_release"
+  warn_and_continue "Failed to get list version on $link_release"
   print=`cat ${TGENV_ROOT}/list_all_versions_offline`
```

Comment:
Hello,

I don't see why the file `list_all_version_offline` is not in the `.gitignore` and updated each time the website is online